javascript/javascript1/week3/Note_taking/notes.js

```diff
@@ -1,35 +1,33 @@
 const notes = [];
-function addNote(content, id) {
-    notes.push({content:content,id:id});
+function addNote(content_in, id_in) {
+    notes.push({content:content_in,id:id_in});
 
 }
-addNote("First Note", 1);//added 1st note
-addNote("Second Note", 2);//added 2nd note
+addNote("First Note", 1);
+addNote("Second Note", 2);
 
 
 function getNoteFromId(id){
-for(let i=0 ;i < notes.length; i++ ){
     if (isNaN(id)){
         console.log("Sorry the entered id is not a number.");
     }
     else{
     index = notes.findIndex(x => x.id === id);
-    const getNotes= notes[index];
-    console.log(getNotes);
-    return getNotes;
+    const noteAtIndex= notes[index];
+    console.log(noteAtIndex);
+    return noteAtIndex;
     }
    
-}
 }
 getNoteFromId(1);
 getNoteFromId(2);
-getNoteFromId("A");// id is not a number
-console.log(notes);// gets all notes.
+getNoteFromId("A");
+console.log(notes);
 
 function logOutNotesFormatted(){
-    for(let i=1 ;i <= notes.length; i++ ){
-        const containedContent = notes[i-1].content;
-        console.log("The note with id:"+ i + ", has the following note text: " + containedContent );
+    for(let i=0 ;i < notes.length; i++ ){
```

Comment:
yes this better and also more conventional, everyone can read this indexing easily.